tests/e2e/sentinel/transitions/multi_report_alerts.spec.js

```diff
@@ -345,7 +345,17 @@ describe('multi_report_alerts', () => {
         time_window_in_days: 1,
         forms: ['FORM']
       }],
-      forms: { FORM: { } }
+      forms: { FORM: { } },
```

Comment:
Please also add an e2e test that doesn't have configuration for `update_clinics`, to assert what happens in those cases. 
I think you can use the same test, but add an additional doc + a new form to submit.

Reply:
A very big oopsie from my side, as I actually meant to add this comment on the `update_clinics` e2e test file, where we actually check the messages and errors that `update_clinics` adds :(